test-project-android/app/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
 
 android {
     compileSdkVersion(33)
+    namespace = "com.kotlinter.example"
```

Comment:
Required by AGP 8.0, setting package name via AndroidManifest results in an error now